src/schema/__tests__/city.test.ts

```diff
@@ -386,23 +387,70 @@ describe("City", () => {
     })
   })
 
-  it("includes sponsored content", async () => {
-    const query = gql`
-      {
-        city(slug: "sacramende-ca-usa") {
-          sponsoredContent {
-            introText
-            artGuideUrl
+  describe("sponsored content", () => {
+    it("includes static texts", async () => {
+      const query = gql`
+        {
+          city(slug: "sacramende-ca-usa") {
+            sponsoredContent {
+              introText
+              artGuideUrl
+            }
           }
         }
+      `
+
+      const result = await runQuery(query)
+
+      expect(result!.city.sponsoredContent).toEqual({
+        introText: "Lorem ipsum dolot sit amet",
+        artGuideUrl: "https://www.example.com/",
+      })
+    })
+
+    it("includes shows and stub shows from a hard-coded list", async () => {
+      const mockShows = [{ id: "sponsored-show" }]
+
+      const mockShowsLoader = jest.fn(() =>
+        Promise.resolve({
+          headers: { "x-total-count": "1" },
+          body: mockShows,
+        })
+      )
+
+      const context = {
+        showsWithHeadersLoader: mockShowsLoader,
       }
-    `
 
-    const result = await runQuery(query)
+      const query = gql`
+        {
+          city(slug: "sacramende-ca-usa") {
+            sponsoredContent {
+              shows(first: 1) {
+                edges {
+                  node {
+                    id
+                  }
+                }
+              }
+            }
+          }
+        }
+      `
 
-    expect(result!.city.sponsoredContent).toEqual({
-      introText: "Lorem ipsum dolot sit amet",
-      artGuideUrl: "https://www.example.com/",
+      const result = await runQuery(query, context)
+      const gravityOptions = context.showsWithHeadersLoader.mock.calls[0][0]
```

Comment:
FYI You can also use Jest’s mocks like so `expect(mock).toHaveBeenCalledWith(…)`